commands/copilot_cli.py

```diff
@@ -16,6 +16,8 @@
 from .utils import mkfile
 from .utils import setup_templates
 
+ADDON_CONFIG_FILENAME = "addons.yml"
```

Comment:
Not sure we need this since it's only used in one place or do we foresee that we will be using this elsewhere?

Reply:
I meant to use this in the test file as it is used a lot. Done now (pending push).

Reply:
Bit of an edge case, but if you use the same constant in the tests, is there is a chance that we could change the value of this constant, break things in the real world, but have all the tests still passing?